### DISCUSSION TITLE
issues in exporting from 'docx' on AWS

### DISCUSSION THREAD

**Discussion by sjengg06** [dolanmiu/docx#1142] [Q&A] [answered]
Hi @dolanmiu,

I'm facing issue in building code that has 'docx' module over aws ecs in a VueJS project. The code runs fine on local but throws following error on AWS ECS (in codebuild). Please let me know if you've any ideas of the reason and fix.

 WARNING  Compiled with 15 warnings11:15:27 PM

 warning  in ./src/components/Test/TestSummary.vue?vue&type=script&lang=js&

**"export 'AlignmentType' was not found in 'docx'**

**Comment by dolanmiu** [comment, marked as answer]
Duplicate of this?

https://github.com/dolanmiu/docx/issues/1155
